executor.py

```diff
@@ -128,7 +133,20 @@ def encode(self, docs: DocumentArray, parameters: Dict[str, Any], **_):
 
     def _encode_images(self, batch: DocumentArray):
         """Encode images using the CLIP image encoder."""
-        tensors_batch = [d.tensor for d in batch]
+        tensors_batch = []
+        for d in batch:
+            if d.blob != b'':
```

Comment:
I think we should first check if there is a tensor and then fall back to blob

Reply:
There can be only blob or tensor because it is a on-of attribute.
So, the order does not pay a role.